client/my-sites/marketing/style.scss

```diff
@@ -77,6 +77,24 @@
 	}
 }
 
+.is-section-marketing {
+	.promo-card:not( .is-primary ) {
+		.action-panel__cta {
+			padding-top: 0;
+		}
+		.action-panel__figure {
+			max-width: 64px;
+		}
+		.action-panel__body p {
+			flex-grow: 0;
```

Comment:
I think this was set to `flex-grow: 1` so that the CTA buttons get anchored to the bottom of the card. Is it related to the Google Ads card text that we need to turn that off?

Reply:
I've just noticed, that I didn't mention you in this comment @davemart-in.  Any thoughts on this?

Reply:
Yeah, I intentionally changed that. The space between the content and the button felt too extreme in some of the cards.

Reply:
That's good, I was just checking it was intended!